### PR TITLE
Update install and config guide for ephemeral storage

### PR DESCRIPTION
**Pull request by ghost**
This is a follow-up to https://github.com/openshift/openshift-docs/pull/10828.

@aheslin 
http://file.rdu.redhat.com/~tmorriso/071718/ic-guide/install_config/configuring_ephemeral.html

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
@jhou1 @liangxia Please provide QE review.

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL.

File: install_config/configuring_ephemeral.adoc [hunk 1]
```diff
@@ -21,26 +21,28 @@ discussed here].
...
 and is disabled by default.
 
-Note that this tech preview does not change any of the mechanisms for
+[NOTE]
+====
+This technology preview does not change any of the mechanisms for
 making local storage available in {product-title}; the existing
-mechanisms (root directory or runtime directory) are still used.  This
-preview only provides mechanism for managing the use of this resource.
+mechanisms (root directory or runtime directory) are still used. This
```
Comment (kalexand-rh) [inline install_config/configuring_ephemeral.adoc]: no parentheses
s/are still used/still apply

File: install_config/configuring_ephemeral.adoc [hunk 2]
```diff
@@ -21,26 +21,28 @@ discussed here].
...
+technology preview only provides a mechanism for managing the use of this resource.
+====
 
 [[ephemeral-storage-enabling-ephemeral-storage]]
-=== Enable Ephemeral Storage
-Enable the `LocalStorageCapacityIsolation` feature gate on all masters and nodes.
+== Enabling ephemeral storage
```
Comment (kalexand-rh) [inline install_config/configuring_ephemeral.adoc]: Please update the case on line 2 and consider making the title on 12 more descriptive.

File: install_config/configuring_ephemeral.adoc [hunk 3]
```diff
@@ -21,26 +21,28 @@ discussed here].
...
-The feature gates are enabled/disabled by means of command line
-arguments to the kubelet on each node (master as well as node).  These
-arguments are passed by means of ConfigMaps in the `openshift-node`
-namespace.  You edit these ConfigMaps via the `oc edit` command.  The
-config maps to be edited may be found via
+To enable ephemeral storage:
 
-```
-oc 
+. Enable the `LocalStorageCapacityIsolation` feature gate on all masters and nodes.
```
Comment (kalexand-rh) [inline install_config/configuring_ephemeral.adoc]: I'd remove this step. It's describing the process you follow in the next steps.

File: install_config/configuring_ephemeral.adoc [hunk 4]
```diff
@@ -21,26 +21,28 @@ discussed here].
...
++
+The feature gates are enabled/disabled by means of command line
+arguments to the kubelet on each node (master as well as node). These
+arguments are then passed by means of ConfigMaps in the `openshift-node`
+namespace. You can edit these ConfigMaps by using the `oc edit` command. 
 
 . Edit or create the master configuration file on all masters (*_/etc/origin/master/master-config.yaml_* by default) and add  `LocalStorageCapacityIsolation=true` under the `apiServerArguments` and `controllerArguments` sections:
```
Comment (kalexand-rh) [inline install_config/configuring_ephemeral.adoc]: No parentheses.
There's an extra space before \`LocalStorageCapacityIsolation=true`.
s/under/in

File: install_config/configuring_ephemeral.adoc [hunk 5]
```diff
@@ -56,45 +58,44 @@ apiServerArguments:
    - LocalStorageCapacityIsolation=true
 ...
 ----
-```
-
-. Edit the ConfigMap for all nodes to enable the LocalStorageCapacityIsolation on the command line.  You can identify the ConfigMaps that need to be edited as follows:
 
-```
+. Edit the ConfigMap for all nodes to enable the LocalStorageCapacityIsolation on the command line. You can identify the ConfigMaps that need to be edited as follows:
```
Comment (kalexand-rh) [inline install_config/configuring_ephemeral.adoc]: I think this needs to use more steps, maybe like:
. Obtain a list of the nodes to add the `LocalStorageCapacityIsolation` parameter to:
+
command & output
. For each node listed, add the `LocalStorageCapacityIsolation` parameter:
.. Edit the ConfigMap:
+
$ oc edit cm node-config-master -n \<openshift-node> \<1>
\<1> \`<openshift_node> is the name of the node to modify.
.. Add the feature gate to the node:
\* If there is already a `feature-gates:` declaration...
\* If there is no `feature-gates:` declaration...

File: install_config/configuring_ephemeral.adoc [hunk 6]
```diff
@@ -56,45 +58,44 @@ apiServerArguments:
...
 node-config-compute   1         52m
 node-config-infra     1         52m
 node-config-master    1         52m
-```
-
-For each of these maps (node-config-compute, node-config-infra, and
-node-config-master), you need to add the featuregate:
-
-```
+----
```
Comment (kalexand-rh) [inline install_config/configuring_ephemeral.adoc]: I'd remove this note.

File: install_config/configuring_ephemeral.adoc [hunk 7]
```diff
@@ -56,45 +58,44 @@ apiServerArguments:
...
-
-Repeat for node-config-compute and node-config-infra, and any other
-ConfigMaps you may find.
+----
 
-Following this, you will need to restart {product-title} and delete the container running the apiserver.
+. Repeat for `node-config-compute`, `node-config-infra`, and any other
+ConfigMaps.
 
+. Restart {product-title} and delete the container running the apiserver.
```
Comment (kalexand-rh) [inline install_config/configuring_ephemeral.adoc]: Show these commands.

File: install_config/configuring_ephemeral.adoc [hunk 8]
```diff
@@ -56,45 +58,44 @@ apiServerArguments:
...
+----
 oc edit cm node-config-master -n openshift-node
-```
-
-If there is already a feature-gates: declaration, you need to add
-
-```
+----
++
+If there is already a `feature-gates:` declaration, you need to add the following to the list of feature gates:
```
Comment (kalexand-rh) [inline install_config/configuring_ephemeral.adoc]: s/you need to add the following/add the following text
(I'd make this part of a ul in the third step.)

File: install_config/configuring_ephemeral.adoc [hunk 9]
```diff
@@ -56,45 +58,44 @@ apiServerArguments:
...
+----
 ,LocalStorageCapacityIsolation=true
-```
-
-to the list of feature gates.  If not, you need to add:
-
-```
+----
++
+If there is no `feature-gates:` declaration, you need to add:
```
Comment (kalexand-rh) [inline install_config/configuring_ephemeral.adoc]: s/you need to add/add the following section under the \<section_this_goes_in>:
(I'd make this part of a ul in the third step.)

File: install_config/configuring_ephemeral.adoc [hunk 10]
```diff
@@ -56,45 +58,44 @@ apiServerArguments:
...
   feature-gates:
   - LocalStorageCapacityIsolation=true
-```
-
-Repeat for node-config-compute and node-config-infra, and any other
-ConfigMaps you may find.
+----
 
-Following this, you will need to restart {product-title} and delete the container running the apiserver.
+. Repeat for `node-config-compute`, `node-config-infra`, and any other
```
Comment (kalexand-rh) [inline install_config/configuring_ephemeral.adoc]: If you take my suggestion about step chunking, remove this step.

**Pull request merged**